Kernel/Storage/BMIDEChannel.h

```diff
@@ -46,12 +46,14 @@ class BMIDEChannel final : public IDEChannel {
 
 public:
     static NonnullRefPtr<BMIDEChannel> create(const IDEController&, IDEChannel::IOAddressGroup, IDEChannel::ChannelType type);
+    static NonnullRefPtr<BMIDEChannel> create(const IDEController&, u8 irq, IDEChannel::IOAddressGroup, IDEChannel::ChannelType type);
```

Comment:
for sake of consistency I would put the irq last, that the rest is symmetric  
same goes for line 56 and the constructor

Reply:
I do see your point, however, there's no real issue with doing it this way.
Also, I want to ensure that the reader of this code clearly understands that this is a different way to create a special `IDEChannel` object, because of the IRQ line being set to something different. It seemed logical to put `u8 irq` just after `const IDEController&` as it is "more important" than the other parameters after it.